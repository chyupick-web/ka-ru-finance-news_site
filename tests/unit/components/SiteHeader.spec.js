import { shallowMount, createLocalVue } from '@vue/test-utils'
import Component from '@/components/SiteHeader.vue'
import BootstrapVue from 'bootstrap-vue'

const localVue = createLocalVue()
localVue.use(BootstrapVue)

describe('SiteHeader.vue', () => {
  it('matches snapshot', () => {
    const wrapper = shallowMount(Component, { localVue })
    expect(wrapper.html()).toMatchSnapshot()
  })
})
