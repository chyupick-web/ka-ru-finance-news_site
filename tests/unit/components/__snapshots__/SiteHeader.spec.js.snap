// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SiteHeader.vue matches snapshot 1`] = `
<header class="bg-cmu-red">
  <div class="container">
    <div class="row">
      <div class="col">
        <h1 class="mt-2 mb-2"><a target="_self" href="https://www.cmu.edu/" class="d-inline-block"><svg class="cmu-wordmark">
              <use xlink:href="#cmu-wordmark"></use>
            </svg></a></h1>
      </div>
    </div>
  </div>
</header>
`;
